app/coffeescripts/jquery/ModuleSequenceFooter.js

```diff
@@ -275,23 +275,42 @@ export default class ModuleSequenceFooter {
   // to display the module name instead of the item title.
   // @api private
 
-  buildNextData () {
-    this.next.url = this.item.next.html_url
+  buildNextPathData (masteryPath) {
+    if (masteryPath.awaiting_choice) {
+      this.next.url = masteryPath.choose_url
+      this.next.tooltipText = I18n.t('Choose the next mastery path')
+    } else {
+      const lockedMessage = I18n.t('Next mastery path is currently locked')
+      const processingMessage = I18n.t('Next mastery path is currently processing')
+      const tooltipText = masteryPath.locked ? lockedMessage : processingMessage
+      this.next.modules_tab_disabled = masteryPath.modules_tab_disabled
+      this.next.url = masteryPath.modules_url
+      this.next.tooltipText = tooltipText
+    }
+    this.next.tooltip = `<i class='${htmlEscape(this.iconClasses.ModuleItem)}'/> ${this.next.tooltipText}`
+  }
 
```

Comment:
@captainhurst, Can you merge these locally and push to this branch for the PR (branch:pr-74).